presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -127,6 +127,7 @@
     private boolean hdfsWireEncryptionEnabled;
 
     private boolean skipDeletionForAlter;
+    private boolean skipTargetCleanupOnFailure;
```

Comment:
Do we wanna make this true by default?

Reply:
In general I'm a fan of having new features off by default to give them a testing period where people opt in to using them. Since this change reduces the amount that Presto cleans up after itself, it could be a problem for people who don't have anything that automatically cleans up their filesystem when nothing is using the data.